src/main/java/TypewiseAlert/CoolingType.java

```diff
@@ -0,0 +1,23 @@
+package TypewiseAlert;
+
+public enum CoolingType {
+	PASSIVE_COOLING(0, 35), HI_ACTIVE_COOLING(0, 45), MED_ACTIVE_COOLING(0, 40);
+
+	int upperLimit;
+	int lowerLimit;
+
+	CoolingType(int upperLimit, int lowerLimit) {
+		// TODO Auto-generated constructor stub

```

Comment:
What does this TODO mean? Do not leave default comments in commits

Reply:
Noted.